GameCore/src/main/java/me/patothebest/gamecore/arena/AbstractArena.java

```diff
@@ -1102,7 +1102,7 @@ public boolean canSpectate(Player player) {
     }
 
     public boolean isFull() {
-        return players.size() >= getMaxPlayers();
+        return players.size() >= maxPlayers;
```

Comment:
The reason for this was so that we could override the getMaxPlayers() method, as seen here: https://github.com/PatoTheBest/MiniGames/blob/master/Games/SkyWars/src/main/java/me/patothebest/skywars/arena/Arena.java#L357
The reason was that we can't have more players than me have spawns in SkyWars or HungerGames and having less players than spawns is illogical.
It's a bad design and I agree with that as it causes confusion, but do you have any suggestions? perhaps updating the maxPlayers manually for these scenarios?
Or would adding a comment on the code suffice?

Reply:
Oh, I didn't notice that Arena was overriding that method, that's why, when setting up my arena, getMaxPlayers was returning 0 (as I didn't have any spawns yet), I believe it causes more harm (confusion) than good, and could be solved easier by checking those things in the canArenaBeEnabled method